app-leechcraft/lc-azoth/lc-azoth-0.6.75_pre20200510.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils leechcraft
+
+DESCRIPTION="Azoth, the modular IM client for LeechCraft"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc +acetamide +adiumstyles astrality +autoidler +autopaste +birthdaynotifier
+	+chathistory +crypt +depester +embedmedia +herbicide +hili +isterique +juick
+	+keeso +lastseen +latex media +metacontacts +murm +nativeemoticons +otroid sarin
+	shx +spell +standardstyles +vader velvetbird +woodpecker +xmpp +xtazy"
+
+REQUIRED_USE="|| ( standardstyles adiumstyles )"
+
+COMMON_DEPEND="
+	~app-leechcraft/lc-core-${PV}
+	dev-qt/qtdbus:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtsql:5
+	dev-qt/qtwebkit:5
+	dev-qt/qtxml:5
+	astrality? ( net-libs/telepathy-qt[qt5(+)] )
+	autoidler? (
+		dev-qt/qtx11extras:5
+		x11-libs/libXScrnSaver
+	)
+	crypt? ( app-crypt/qca:2[qt5(+)] )
+	lastseen? ( dev-qt/qtconcurrent:5 )
+	media? ( dev-qt/qtmultimedia:5 )
+	otroid? (
+		dev-qt/qtconcurrent:5
+		net-libs/libotr
+	)
+	sarin? (
+		dev-qt/qtconcurrent:5
+		net-libs/tox:=
+	)
+	woodpecker? ( dev-libs/kqoauth )
+	xmpp? (
+		>=net-libs/qxmpp-1.2.0
+		media? ( net-libs/qxmpp[speex] )
+	)
+	xtazy? ( ~app-leechcraft/lc-xtazy-${PV} )
+"
+DEPEND="${COMMON_DEPEND}
+	doc? ( app-doc/doxygen[dot] )
```

Comment:
This is a BDEPEND, which then makes `COMMON_DEPEND` redundant here (just call it DEPEND).